### PR TITLE
Manage EBS Volumes

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -10,3 +10,4 @@
 
 depends 'ohai'
 depends 'et_fog', '>= 1.1.0'
+depends 'aws', '~> 3.3'
```
Comment (jeffbyrnes) [inline metadata.rb]: I realize this is beyond the scope of this work in a massive way, but I’d love to deprecate `et_fog` in favor of using the `aws` gem.

File: recipes/ec2test.rb [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-end.run_action(:create)
-
-file '/etc/chef/ohai/hints/ec2.json' do
-  content JSON.pretty_generate({})
-  action :nothing
-end.run_action(:create)
-
-ohai 'ec2_hints' do
-  action :nothing
-end.run_action(:reload)
```
Comment (jeffbyrnes) [inline recipes/ec2test.rb]: This appears to have snuck in here; this is old code from when you have to do this to get Ohai to properly populate on first boot, right?

**Comment by jeffbyrnes** [comment]
This looks good; just a question & a thought for future work.

File: metadata.rb (hunk 1 above)
Comment (eherot) [inline metadata.rb]: I'm assuming you're actually talking about `aws-sdk`. And yes I'd agree but...out of scope.

File: recipes/ec2test.rb (hunk 2 above)
Comment (eherot) [inline recipes/ec2test.rb]: Yeah fixing this now.

File: metadata.rb (hunk 1 above)
Comment (jeffbyrnes) [inline metadata.rb]: Sorry, yes, I am. Esp. since `chef-provisioning`, and AWS itself, has standardized on it.

**Pull request merged**